app/addons/components/tests/stringEditModalSpec.react.jsx

```diff
@@ -86,11 +88,13 @@ define([
         );
 
         var modal = TestUtils.findRenderedComponentWithType(el, Modal);
-        var modalEl = React.findDOMNode(modal.refs.modal);
+        var modalEl = ReactDOM.findDOMNode(modal.refs.modal);
 
         TestUtils.Simulate.click($(modalEl).find('#string-edit-save-btn')[0]);
         assert.ok(spy.calledOnce);
```

Comment:
this test makes no sense and has to get fixed

test title is: "replaces "\\n" with actual newlines"

the actual test asserts that a spy was called once, nothing more.